.changeset/swift-carpets-change.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/netlify': patch
```

Comment:
There’s maybe an argument this is a breaking change and should have been a major version? But happy to defer to you on that.